src/org/rapla/plugin/mail/server/MailapiClient.java

```diff
@@ -36,15 +40,6 @@ public MailapiClient( RaplaContext context) throws  RaplaException {
     public MailapiClient()
     {
     }
-    
-    public boolean isSsl() {
-		return ssl;
-	}
-
-	public void setSsl(boolean ssl) {
-		this.ssl = ssl;
-	}
-
```

Comment:
These getter/setter were called nowhere explicitly, so I removed them. Is there any hidden cases where they are actually called (through some serializations, for example)